src/helpers/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
We're using `normalize-css-color` but, in order to get proper type support, I copied the repo source over here and added type definitions / some ES6 sugar it was missing. It seemed from their license that we needed to include the LICENSE file in source. cc/ @ryan-roemer any thoughts here on what other steps we should take? Here's the original repo: https://github.com/react-native-community/normalize-css-color `react-spring` seems to do this as well although the license text is in file here: https://github.com/react-spring/react-spring/blob/master/src/shared/normalizeColors.ts